contentcuration/contentcuration/frontend/channelList/views/ChannelSet/StudioCollectionsTable.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+
+  <KPageContainer class="page-container">
+    <div class="table-header">
+      <div class="header-left">
+        <KButton
+          appearance="basic-link"
+          class="info-link"
+          @click="infoDialog = true"
+        >
+          {{ $tr('aboutChannelSetsLink') }}
+        </KButton>
+
+        <KModal
+          v-if="infoDialog"
+          :title="$tr('aboutChannelSets')"
+          :cancelText="$tr('cancelButtonLabel')"
+          @cancel="infoDialog = false"
+        >
+          <div class="info-content">
+            <p>{{ $tr('channelSetsDescriptionText') }}</p>
+            <p>{{ $tr('channelSetsInstructionsText') }}</p>
+            <p
+              class="disclaimer"
+              :style="{ color: $themeTokens.error }"
+            >
+              {{ $tr('channelSetsDisclaimer') }}
+            </p>
+          </div>
+        </KModal>
+      </div>
+
+      <div class="header-right">
+        <KButton
+          v-if="!loading"
+          appearance="raised-button"
+          primary
+          data-test="add-channelset"
+          :text="$tr('addChannelSetTitle')"
+          @click="newChannelSet"
+        />
+      </div>
+    </div>
+
+    <div class="table-container">
+      <KTable
+        class="collections-table"
+        :stickyColumns="stickyColumns"
+        caption=""
+        :headers="tableHeaders"
+        :rows="tableRows"
+        :dataLoading="loading"
+        sortable
+        disableBuiltinSorting
+        :emptyMessage="$tr('noChannelSetsFound')"
```

Comment:
We've recently (https://github.com/learningequality/studio/pull/5543) updated link position to be within the empty message:

<img width="400" height="auto" alt="" src="https://github.com/user-attachments/assets/1dc18f77-cec3-4e6d-af2d-7856c9085643" />

Would you update this file to achieve similar behavior (but within the new white page container)?

Presence of link in the empty message text will mean that we won't be using `emptyMessage` prop but implement the empty message logic here in `StudioCollectionsTable` - this should align nicely together with the other suggestion for loading state improvement.